src/scss/_theme.scss

```diff
@@ -70,6 +70,13 @@ $boolean-padding: $spacing-double;
 
   // Universal
   --a-white: #fff;
+
+  // Apostrophe brand
```

Comment:
You didn't use these here. It'd be better to add them in a separate commit or in one where you are using them.